example/predictgnn.json

```diff
@@ -1,8 +1,7 @@
 {
   "py/object": "dfpl.options.GnnOptions",
```

Comment:
Can you verify that the content of the example files is identical to the state before https://github.com/yigbt/deepFPlearn/pull/22/

Feel free to add additional test data in independent files (maybe even structured in directories). Then add the corresponding test calls to the `pr.yaml` file.

Reply:
Hooray!all passed!